src/Paramore.Brighter.Extensions.DependencyInjection/IBrighterBuilder.cs

```diff
@@ -81,20 +85,20 @@ public interface IBrighterBuilder
         /// </summary>
         /// <param name="assemblies">The assemblies to scan</param>
         /// <returns>This builder, allows chaining calls</returns>
-        IBrighterBuilder MapperRegistryFromAssemblies(params Assembly[] assemblies);
```

Comment:
Why did you replace the params with IEnumerable? I think during setup, use the params is very helpful

Reply:
Because I needed additional parameters for registering handler types that use dynamic routing, and I wanted that list to be optional. As params needs to be last, that would mean the exclusion list preceded the assembly list, which feels like a poor API design. So I removed the params because we need more than just the params